project/script_IR.tcl

```diff
@@ -12,7 +15,11 @@ set_top InputRouterTop
 add_files ../TrackletAlgorithm/InputRouterTop.cc -cflags "$CFLAGS"
 add_files -tb ../TestBenches/InputRouter_test.cpp -cflags "$CFLAGS"
 
-open_solution "solution1"
+if { ([string first "vitis" $exe] > -1) && ($year > 2019) } {
```

Comment:
@aperloff Is it possible to define something like an alias of `open_solution` in env_hls.tcl that basically adds the `-flow_target` option when appropriate? It would be nice if we could avoid duplicating this if-else block in each script.

Reply:
Based on https://wiki.tcl-lang.org/page/interp+alias I think this is possible. Let me look into it and get back to you.